assets/js/vue/utils/template-switch.js

```diff
@@ -0,0 +1,26 @@
+const grp = {
+    status: 'Dataset Status',
+    fundingCycle: 'Grant Awards',
+    researchGroup:  'Projects',
+    fundingOrg: 'Funding Organizations'
+};
+
+const griidc = {
+    status: 'Dataset Status',
+    fundingCycle: 'Funding Cycles',
+    researchGroup:  'Research Groups',
+    fundingOrg: 'Funding Organizations'
+}
+
+export default {
+    getLabel: (name) => {
+        if (PELAGOS_TEMPLATE_PROPS === undefined || PELAGOS_TEMPLATE_PROPS.BaseTemplateName === undefined || PELAGOS_TEMPLATE_PROPS.BaseTemplateName === 'GRIIDC') {
+            return griidc[name];
+        } else {
+            return grp[name];
+        }
+    },
+    isGrpTemplate: () => {
+        return PELAGOS_TEMPLATE_PROPS.BaseTemplateName === 'GRP';
```

Comment:
This will fail is `PELAGOS_TEMPLATE_PROPS` is undefined.

Reply:
Also see comment above.

Reply:
Do we really need to insure it is set? The templates for griidc and grp set it, and they are not external parameters. How far should we go?

Reply:
Yes, we will probably have more templates, these template properties should be optional in my mind. So one doesn't have to code for every scenario.

Reply:
I'm getting rid of this and instead created a flag section I added to the template-switch.js utility. I think it'll be cleaner, but we always have to handle the undefined cases.